packages/input-number/src/input-number.vue

```diff
@@ -93,6 +93,10 @@
         default: -Infinity
       },
       value: {},
+      sensitivity: {
+        type: Number,
+        default: 100
+      },
```

Comment:
Added `sensitivity` prop to allow the "scrolling" behavior to be configured to be faster or slower as needed.